sdk/javascript/packages/core/internal/quicktest.ts

```diff
@@ -23,20 +25,21 @@ const oneTimeToken = 'US:ONE_TIME_TOKEN'
 
 async function test() {
     const kvs = localConfigStorage(configFileName)
-    await initializeStorage(kvs, oneTimeToken)
+    //await initializeStorage(kvs, oneTimeToken)
     const options: SecretManagerOptions = {
         storage: kvs,
         // queryFunction: cachingPostFunction
         allowUnverifiedCertificate: true
     }
     const { records } = await getSecrets(options)
-    // const { records } = await getSecrets(options, ['SECRET_UID'])
+    //const { records } = await getSecrets(options, ['SECRET_UID'])
     console.log(inspect(records, false, 6))
 
-    // const templateRecord = records[1]
-    // templateRecord.data.title = 'RF14'
+    //const templateRecord = records[1]
+    //templateRecord.data.title = 'RF14'
     // const recordUid = await createSecret(options, templateRecord.folderUid!, templateRecord.data)
     // console.log(recordUid)
+    //await deleteSecret(options, ["fcvDivuK72R-VneHRTMXVw","9_EOpc2wpGkrIXaPOahsWQ"])
```

Comment:
Same, don't leave real rec uids here. Not a secret, but makes bad impression for open source. Undo this file.